apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -201,6 +201,21 @@
       </div>
 
     </statusbar>
+
+    <portal to="modals">
+      <modal class="vue-dialog beekeeper-modal" :name="`discard-changes-modal-${tab.id}`">
+        <div class="dialog-content">
+          <div class="dialog-c-title">Confirmation</div>
+          <div class="modal-form">
+            Sorting or Filtering now will discard {{pendingChangesCount}} pending change(s). Are you sure?
+          </div>
+        </div>
+        <div class="vue-dialog-buttons">
+          <button class="btn btn-flat" type="button" @click.prevent="$modal.hide(`discard-changes-modal-${tab.id}`)">Cancel</button>
```

Comment:
Also unrelated: Using the name is so ugly :vomiting_face:

Reply:
Agreed, I hate it 😂.